tests/darwin/dataset/remote_dataset_test.py

```diff
@@ -378,6 +378,28 @@ def it_works(darwin_client: Client, dataset_name: str, dataset_slug: str, team_s
         assert item_1.id == 386074
         assert item_2.id == 386073
 
+    @responses.activate
+    def it_fetches_files_with_commas(
+        darwin_client: Client, dataset_name: str, dataset_slug: str, team_slug: str, files_content: dict
+    ):
+        remote_dataset = RemoteDataset(
+            client=darwin_client, team=team_slug, name=dataset_name, slug=dataset_slug, dataset_id=1
+        )
+        url = "http://localhost/api/datasets/1/items?page%5Bsize%5D=500"
+        responses.add(
+            responses.POST,
+            url,
+            json=files_content,
+            status=200,
+        )
+
+        actual = remote_dataset.fetch_remote_files({"filenames": ["example,with, comma.mp4"]})
+        list(actual)
```

Comment:
and we don't need `actual`, so we should be fine with this anyway
```suggestion
        list(remote_dataset.fetch_remote_files({"filenames": ["example,with, comma.mp4"]}))
```